appservice/src/intTest/groovy/nu/yona/server/BasicBuddyTest.groovy

```diff
@@ -132,11 +134,16 @@ class BasicBuddyTest extends Specification {
 
 		when:
 			def response = appService.getDirectMessages(bobDunnURL, bobDunnPassword)
-			bobDunnBuddyMessageAcceptURL = response.responseData._embedded.buddyConnectRequestMessages[0]._links.accept.href
+			if (response.responseData._embedded && response.responseData._embedded.buddyConnectRequestMessages) {
+				bobDunnBuddyMessageAcceptURL = response.responseData._embedded.buddyConnectRequestMessages[0]._links.accept.href
+			}
 
 		then:
 			response.status == 200
 			response.responseData._links.self.href == bobDunnURL + appService.DIRECT_MESSAGE_PATH_FRAGMENT
+			response.responseData._embedded
```

Comment:
The test results already shows the per path fragment what its value is, so this is not necessary